proxy.go

```diff
@@ -121,7 +121,7 @@ func newProxy(
 	bundle.Check()
 	// TODO: make circuit breaker configurable
 	circuitBreaker := cb.NewCircuitBreaker(
-		cb.MaxRetries(math.MaxInt64),
+		cb.MaxRetries(validationutil.MaxInt),
```

Comment:
How about using `math.MaxInt32` here?

Reply:
I couldn't follow the reasoning, can you explain further? I think MaxInt let us use the platform specific maximum value here

Reply:
Even if the backoff time is 10 milliseconds, then with the range of `math.MaxInt32` the retries would last for 120 days. I think we can assume that's infinite. having `math.MaxInt32` solves the problem, and I think it's the simplest way to fix that. What do you think?